scripts/update-ts-runtime.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+#
+# This script will update the treesitter runtime to the provided commit.
+# Usage :
+#   $0 <tree-sitter commit sha>
+
+ts_source_dir="/tmp/tree-sitter"
+ts_url="https://github.com/tree-sitter/tree-sitter.git"
+
+base_dir="$(cd "$(dirname $(dirname $0))" && pwd)"
+ts_dest_dir="$base_dir/src/tree_sitter/"
+
+echo "$ts_dest_dir"
+
+if [ ! -d "$ts_source_dir" ]; then
+  echo "Cloning treesitter..."
+  git clone "$ts_url" "$ts_source_dir"
+else
+  echo "Found a non-empty $ts_source_dir directory..."
+fi
+
+echo "Checking out $1..."
+cd "$ts_source_dir"
```

Comment:
Do you need to `cd` here?

Reply:
That's something left from an earlier version, not used anymore, you are right.